sdk/push-fcm/src/main/java/com/klaviyo/pushFcm/KlaviyoNotification.kt

```diff
@@ -124,7 +139,43 @@ class KlaviyoNotification(private val message: RemoteMessage) {
             .setNumber(message.notificationCount)
             .setPriority(message.notificationPriority)
             .setAutoCancel(true)
-            .build()
+
+    private fun URL.applyToNotification(builder: NotificationCompat.Builder) {
```

Comment:
If we did want to expose a helper method for downloading and applying the image, all we'd have to do is make this extension method public @ajaysubra circling back to yesterday's conversation.